src/ig/IGExporter.ts

```diff
@@ -1338,96 +1342,169 @@ export class IGExporter {
     }
   }
 
+  // Supports necessary updates for R5 IGs.
+  // SUSHI only supports R4 or later, and as of now, we will just target R5.
   updateForR5(): void {
-    // If it isn't R4, we will update it. SUSHI only supports R4 or later, and as of now, we will just target R5.
-    if (!isR4(this.config.fhirVersion)) {
-      // Update IG.definition.resource
-      this.ig.definition.resource.forEach(resource => {
-        // Use IG.definition.resource.isExample
-        if (resource.exampleBoolean || resource.exampleCanonical) {
-          resource.isExample = true;
-          if (resource.exampleCanonical) {
-            resource.profile = [resource.exampleCanonical];
-          }
-          delete resource.exampleBoolean;
-          delete resource.exampleCanonical;
-        } else if (resource.exampleBoolean === false) {
-          resource.isExample = false;
-          delete resource.exampleBoolean;
+    // Update IG.definition.resource
+    this.ig.definition.resource.forEach(resource => {
+      // Use IG.definition.resource.isExample
+      if (resource.exampleBoolean || resource.exampleCanonical) {
+        resource.isExample = true;
+        if (resource.exampleCanonical) {
+          resource.profile = [resource.exampleCanonical];
         }
+        delete resource.exampleBoolean;
+        delete resource.exampleCanonical;
+      } else if (resource.exampleBoolean === false) {
+        resource.isExample = false;
+        delete resource.exampleBoolean;
+      }
```

Comment:
I'd recommend either changing the settings on the Files tab or using `-w` in the command line because wow I hadn't looked at the UI since making that change it is really is rough.